<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="ProfileAssignmentsBusiness">
  <testsuite name="Login to CB as primaryadmin" id="6f813ae1-cff6-4c3a-ab16-31fe2662981e" tests="1" time="1084">
    <testcase name="status code is 201 : Created"/>
  </testsuite>
  <testsuite name="Add Business Profile" id="908ad05a-4393-4cb8-8471-0300485afe5c" tests="0" time="0">
    <error>
      <![CDATA[Iteration: 0
RequestError: Error: read ECONNRESET
    at exports._errnoException (util.js:907:11)
    at TLSWrap.onread (net.js:557:26)

---
testScriptError: ReferenceError: responseBody is not defined
    at Object.eval (eval at <anonymous> (evalmachine.<anonymous>:284:34), <anonymous>:3:27)
    at evalmachine.<anonymous>:293:23
    at evalmachine.<anonymous>:299:2
    at Object.exports.runInContext (vm.js:44:17)
    at _.extend._run (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:156:16)
    at _.extend.execute (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:121:14)
    at PostmanCollectionRun.<anonymous> (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\runner\extensions\event.command.js:179:27)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1144:13
    at replenish (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1018:21)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1022:13
---
]]>
    </error>
  </testsuite>
  <testsuite name="get profile by Id copy" id="24fa06a8-95ec-4772-842c-eca442cde3e4" tests="1" time="797">
    <error>
      <![CDATA[Iteration: 0

---
testScriptError: TypeError: Cannot read property 'ChannelProfiles' of undefined
    at Object.eval (eval at <anonymous> (evalmachine.<anonymous>:284:34), <anonymous>:5:21)
    at evalmachine.<anonymous>:293:23
    at evalmachine.<anonymous>:299:2
    at Object.exports.runInContext (vm.js:44:17)
    at _.extend._run (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:156:16)
    at _.extend.execute (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:121:14)
    at PostmanCollectionRun.<anonymous> (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\runner\extensions\event.command.js:179:27)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1144:13
    at replenish (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1018:21)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1022:13
---
]]>
    </error>
    <testcase name="status code is 200 : OK">
      <failure type="AssertionFailure">
        <![CDATA[Failed 1 times.]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="business user add with profile" id="7f57ebed-71a9-48a3-a177-a34367055a58" tests="1" time="667">
    <error>
      <![CDATA[Iteration: 0

---
testScriptError: TypeError: Cannot read property 'Id' of undefined
    at Object.eval (eval at <anonymous> (evalmachine.<anonymous>:284:34), <anonymous>:5:51)
    at evalmachine.<anonymous>:293:23
    at evalmachine.<anonymous>:299:2
    at Object.exports.runInContext (vm.js:44:17)
    at _.extend._run (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:156:16)
    at _.extend.execute (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:121:14)
    at PostmanCollectionRun.<anonymous> (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\runner\extensions\event.command.js:179:27)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1144:13
    at replenish (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1018:21)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1022:13
---
]]>
    </error>
    <testcase name="status code is 201 : Created">
      <failure type="AssertionFailure">
        <![CDATA[Failed 1 times.]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="Get Business Users with profile" id="623887df-4347-4c12-9b4e-eb19a12d6f5d" tests="0" time="638">
    <error>
      <![CDATA[Iteration: 0

---
testScriptError: TypeError: Cannot read property 'UserName' of undefined
    at Object.eval (eval at <anonymous> (evalmachine.<anonymous>:284:34), <anonymous>:4:53)
    at evalmachine.<anonymous>:293:23
    at evalmachine.<anonymous>:299:2
    at Object.exports.runInContext (vm.js:44:17)
    at _.extend._run (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:156:16)
    at _.extend.execute (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:121:14)
    at PostmanCollectionRun.<anonymous> (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\runner\extensions\event.command.js:179:27)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1144:13
    at replenish (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1018:21)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1022:13
---
]]>
    </error>
  </testsuite>
  <testsuite name="modify business user" id="31f50ad8-faaf-45c6-9bfc-2e5ebf8e87db" tests="1" time="641">
    <testcase name="status code is 204 : No Content">
      <failure type="AssertionFailure">
        <![CDATA[Failed 1 times.]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="Get Business Users with profile copy" id="963cab32-0d07-46b9-8c09-3cf83d22a536" tests="0" time="1106">
    <error>
      <![CDATA[Iteration: 0

---
testScriptError: TypeError: Cannot read property 'UserName' of undefined
    at Object.eval (eval at <anonymous> (evalmachine.<anonymous>:284:34), <anonymous>:4:53)
    at evalmachine.<anonymous>:293:23
    at evalmachine.<anonymous>:299:2
    at Object.exports.runInContext (vm.js:44:17)
    at _.extend._run (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:156:16)
    at _.extend.execute (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:121:14)
    at PostmanCollectionRun.<anonymous> (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\runner\extensions\event.command.js:179:27)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1144:13
    at replenish (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1018:21)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1022:13
---
]]>
    </error>
  </testsuite>
  <testsuite name="business user add without profile" id="a2a35188-22af-4f5c-ab31-8612a2732cb6" tests="1" time="674">
    <testcase name="Profile is required."/>
  </testsuite>
  <testsuite name="business user add with channel profile" id="fe8abdd5-b08a-4ee2-8418-32f4a16af3c6" tests="1" time="669">
    <testcase name="Profile ID is Invalid.">
      <failure type="AssertionFailure">
        <![CDATA[Failed 1 times.]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="business user add with retailUser profile" id="efe2121b-ce2d-4702-874f-6ba8938495e9" tests="1" time="660">
    <testcase name="Profile ID is Invalid."/>
  </testsuite>
  <testsuite name="business user add with invalid profileId" id="98262d2f-6a2e-4788-b5dd-4190a1f7078a" tests="1" time="699">
    <testcase name="Profile ID is Invalid."/>
  </testsuite>
  <testsuite name="business user add with multiple profile" id="7d8175fb-bbdc-47b5-99ef-39fd093569e1" tests="1" time="1939">
    <testcase name="User can be assigned to only one Profile."/>
  </testsuite>
  <testsuite name="Login to CB as jolina" id="ca8ed332-6fb3-4592-a235-52a26d946264" tests="1" time="876">
    <testcase name="status code is 201 : Created"/>
  </testsuite>
  <testsuite name="Add Business Profile with user not entitled to add" id="e52bfa80-8875-4d46-9647-07a77d6707e8" tests="1" time="267">
    <testcase name="status code is 400 : Bad Request"/>
  </testsuite>
  <testsuite name="business user add with profile with user not entitled" id="7f82e1c7-e809-4370-ab06-d4056e2b0b7d" tests="1" time="593">
    <testcase name="status code is 400 : Bad Request"/>
  </testsuite>
  <testsuite name="Login to CB as secondaryadmin" id="b68a4fde-b7c7-4aed-b62a-f31c3c043635" tests="1" time="802">
    <testcase name="status code is 201 : Created"/>
  </testsuite>
  <testsuite name="Add Business Profile with secondaryadmin" id="2c5807bc-829a-407d-9e1d-e36392173e31" tests="1" time="335">
    <testcase name="status code is 201 : Created"/>
  </testsuite>
  <testsuite name="business user add with secondaryadmin" id="0539da73-6143-428d-b78d-e2e1e1014ea8" tests="1" time="1875">
    <testcase name="status code is 201 : Created"/>
  </testsuite>
  <testsuite name="delete assigned profile" id="16a4e1fe-d4d0-4fb3-9002-1f34c5874a8b" tests="0" time="0">
    <error>
      <![CDATA[Iteration: 0
RequestError: Error: read ECONNRESET
    at exports._errnoException (util.js:907:11)
    at TLSWrap.onread (net.js:557:26)

---
testScriptError: ReferenceError: responseBody is not defined
    at Object.eval (eval at <anonymous> (evalmachine.<anonymous>:284:34), <anonymous>:3:45)
    at evalmachine.<anonymous>:293:23
    at evalmachine.<anonymous>:299:2
    at Object.exports.runInContext (vm.js:44:17)
    at _.extend._run (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:156:16)
    at _.extend.execute (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\uvm\host.js:121:14)
    at PostmanCollectionRun.<anonymous> (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\lib\runner\extensions\event.command.js:179:27)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1144:13
    at replenish (C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1018:21)
    at C:\Users\I329272\AppData\Roaming\npm\node_modules\newman\node_modules\postman-runtime\node_modules\async\dist\async.js:1022:13
---
]]>
    </error>
  </testsuite>
</testsuites>